src/services/walktroughServices.js

```diff
@@ -0,0 +1,109 @@
+import { create, watch, currentUser, OpenShiftWatchEvents } from './openshiftServices';
```

Comment:
The filename for `walktroughServices.js` is missing an h => `walkthroughServices.js`